### PR TITLE
:construction_worker: circle ci

### LINKED ISSUE #5: circleci

### PR DESCRIPTION
**Pull request by fabienjuif**
fixes #5

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node
+    steps:
```
Comment (delphinemillet) [inline .circleci/config.yml:6]: can you please explain me `restore_cache` and `save_cache` steps ?

Comment (fabienjuif) [reply .circleci/config.yml:6]: A cache has an id, here the id is `cache-{{ checksum "yarn.lock" }}` (which means that we change all the cache when the yarn.lock file change (if we add/remove/update dependencies))

We can have fallbacks ids
 - if "cache-128092189028190" is not found, we can fallback to the next one

Here I choose not to have fallbacks, but a classic pattern is to have
```
      - restore_cache:
          keys:
          - cache-{{ checksum "yarn.lock" }}
          - cache-
```

-> If i don't find "cache-feiuezfzi" I take the next cache that match "cache-".


 - retore_cache is were we look after a cache to restore with the previous description I give, you can have multiples keys
 - save_cache is where you save a cache for a unique id

Comment (fabienjuif) [reply .circleci/config.yml:6]: Here we choose to cache the dependencies (node_modules) and the yarn cache.
So each time we run a new build, yarn local cache and node_modules are not empty and we don't need to download all dependencies everytime

Comment (delphinemillet) [reply .circleci/config.yml:6]: very clear, thanks :ok_hand:

**Review by delphinemillet — APPROVED** [review]

**Pull request merged**